lib/ansible/modules/cloud/amazon/kms.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {
+  'version': '1.0',
+  'status': ['preview'],
+  'supported_by': 'committer'
+}
+
+DOCUMENTATION = '''
+---
+module: kms
```

Comment:
Can you rename this as `aws_kms` please? Three-letter acronyms have enough risk for overlap that it's probably worth doing.